tests/unit/helpers/format-message-test.js

```diff
@@ -179,7 +179,7 @@ test('able to discover all register translations', function(assert) {
   assert.expect(1);
   this.intl.addTranslation('es_MX', 'foo', 'bar'); /* tests that the locale name becomes normalized to es-mx */
   this.intl.exists('test', 'fr-ca');
-  assert.equal(this.intl.get('locales').join('; '), 'en-us; es-es; fr-fr; es-mx');
```

Comment:
correct me if I'm wrong: we usually know the available locales at build-time, what's keeping us from passing that information to the runtime service? in the case of sideloading translations that are not compiled by us we could offer a property somewhere that you could overwrite at build-time (e.g. by traversing a file system structure).